src/strigo/scripts/configs/__init__.py

```diff
@@ -1,15 +1,13 @@
 # coding: utf8
 from __future__ import annotations
 
+import json
```

Comment:
replace global imports by specific imports: `from json import dumps`

- it gives the imports section more meaningful about goes on in the script (helps collaboration: when one opens a file, one has a better idea about what the script is likely to do)
- it clutters the module's memory a little bit less (less references to other things)
- it shortens the script instructions

Reply:
Good point.
But I'll make a global pass on this later.